libreyolo/cli/commands/label.py

```
@@ -0,0 +1,90 @@
+    host: str = typer.Option("127.0.0.1", help="Host/interface to bind"),
+    port: int = typer.Option(8000, help="Port to bind (auto-bumps if taken)"),
+    device: str = typer.Option("auto", help="Device for AI auto-label: 0, cpu, mps, auto"),
+    no_assist: bool = typer.Option(
+        False, "--no-assist", help="Disable AI auto-label (pure manual labeller)"
+    ),
+    no_browser: bool = typer.Option(
+        False, "--no-browser", help="Do not auto-open the browser"
+    ),
+    json_output: bool = typer.Option(False, "--json", help="JSON output to stdout"),
+    quiet: bool = typer.Option(False, "--quiet", help="Suppress stderr"),
+    verbose: bool = typer.Option(False, "--verbose", help="Verbose stderr output"),
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)